packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogRumPluginTest.kt

```diff
@@ -737,7 +731,7 @@ class DatadogRumPluginTest {
         verify { mockResult.success(null) }
     }
 
-    @Test
+    @Test()
```

Comment:
brackets are redundant with this annotation

```suggestion
    @Test
```

Reply:
Yeup, sorry, I made it repeating to find the error and forgot to remove the parens.